### PR TITLE
[styles] Added icon for monorail stations

### PR DESCRIPTION
**Pull request by ubahnverleih**
Monorail stations were already in the `mapcss-mapping.csv` but had no dedicated icon.
I created the icons based on the already existing `transit_monorail-m.svg` icon used in transit navigation. I modified the shape to fit the pixel rasters so it looks better in these low resolution map icons.
I also added `[railway=halt][station=monorail]` to `railway|station|monorail` in `mapcss-mapping.csv` as it's quite often used on monorails, but for map users the difference between station and halt shouldn't matter.

### CODE REVIEW

File: android/app/src/main/res/values-de/strings.xml
Comment (pastk) [inline android/app/src/main/res/values-de/strings.xml:1]: please put regenerated strings into a separate "[strings] Regenerate" commit

File: data/styles/default/dark/symbols/monorail-m.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg width="100%" height="100%" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;">
```
Comment (pastk) [inline data/styles/default/dark/symbols/monorail-m.svg:3]: please add `width="18" height="18"` (helps to preview in real size)
also maybe cleanup unnecessary tags a bit, see e.g. `bus-m.svg`
:pray:

**Review by pastk — COMMENTED** [review]
Looks good, just small nits!

File: android/app/src/main/res/values-de/strings.xml
Comment (ubahnverleih) [reply android/app/src/main/res/values-de/strings.xml:1]: Done

File: data/styles/default/dark/symbols/monorail-m.svg (hunk 1 above)
Comment (ubahnverleih) [reply data/styles/default/dark/symbols/monorail-m.svg]: fixed width and height values, removed unnecessary attributes and and DOCTYPE tag

**Comment by ubahnverleih** [comment]
OK, fixed both things.

**Review by pastk — APPROVED** [review]
LGTM, many thanks!

Please rebase the styles regeneration again, sometimes its #6167, sorry!

**Comment by pastk** [comment]
@vng @biodranik

**Comment by pastk** [comment]
> Any advice on how i should handle parallel PRs for styles?

Unfortunately the only currently working way now is to rebase a PR after another style PR had been merged..
Fortunately id doesn't happen very often.

**Review by vng — APPROVED** [review]

**Pull request merged**